src/views/PurchaseOrderDetail.vue

```diff
@@ -72,7 +72,8 @@
             </div>
 
             <div class="qty-progress">
-              <ion-progress-bar :value="item.quantityAccepted/item.quantity" />
+              <!-- TODO: improve the handling of quantityAccepted -->
+              <ion-progress-bar :color="parseInt(item.quantityAccepted)/(item.quantity - getPOItemAccepted(item.productId)) > 1 ? 'danger' : 'primary'" :value="(parseInt(item.quantityAccepted) +  getPOItemAccepted(item.productId))/(item.quantity)" />
```

Comment:
We can create a computed property instead of defining logic in template.

Reply:
Made the requested change